### DISCUSSION TITLE
Why use mitosis compile instead of mitosis build?

### DISCUSSION THREAD

**Discussion by kingzez** [CKGrafico/papanasi#48] [Q&A] [answered]
Why use mitosis compile instead of mitosis build?
Any consideration for this? Just to do specific processing for different frameworks?

**Comment by CKGrafico** [comment, marked as answer]
Very good question, I've decided this because (at least when I started) mitosis compile generates JavaScript transpiled files and build only compiles to each different framework and this let me to modify whatever I need and decide how to transpile to ES5 or whatever I need.
